mibidata/tests/test_tiff.py

```diff
@@ -25,15 +25,30 @@
 CHANNELS = ((1, 'Target1'), (2, 'Target2'), (3, 'Target3'),
             (4, 'Target4'), (5, 'Target5'))
 METADATA = {
-    'run': 'Run', 'date': '2017-09-16T15:26:00',
-    'coordinates': (12345, -67890), 'size': 300., 'slide': '857',
+    'run': '20180703_1234_test', 'date': '2017-09-16T15:26:00',
+    'coordinates': (12345, -67890), 'size': 500., 'slide': '857',
+    'fov_id': 'Point1', 'fov_name': 'R1C3_Tonsil',
+    'folder': 'Point1/RowNumber0/Depth_Profile0',
+    'dwell': 4, 'scans': '0,5', 'aperture': '300um',
+    'instrument': 'MIBIscope1', 'tissue': 'Tonsil',
+    'panel': '20170916_1x', 'version': '1.0', 'mass_offset': 0.1,
+    'mass_gain': 0.2, 'time_resolution': 0.5, 'miscalibrated': False,
+    'check_reg': False, 'filename': '20180703_1234_test',
+    'description': 'test image'
+}
+USER_DEFINED_METADATA = {'x_size': 500., 'y_size': 500., 'mass_range': 20}
+OLD_METADATA = {
+    'run': '20180703_1234_test', 'date': '2017-09-16T15:26:00',
+    'coordinates': (12345, -67890), 'size': 500., 'slide': '857',
     'point_name': 'R1C3_Tonsil', 'dwell': 4, 'scans': '0,5',
     'folder': 'Point1/RowNumber0/Depth_Profile0',
     'aperture': '300um', 'instrument': 'MIBIscope1', 'tissue': 'Tonsil',
-    'panel': '20170916_1x', 'mass_offset': 0.1, 'mass_gain': 0.2,
-    'time_resolution': 0.5, 'miscalibrated': False, 'check_reg': False,
-    'filename': 'Run'
+    'panel': '20170916_1x', 'version': None, 'mass_offset': 0.1,
+    'mass_gain': 0.2, 'time_resolution': 0.5, 'miscalibrated': False,
+    'check_reg': False, 'filename': '20180703_1234_test'
 }
+OLD_TIFF_FILE = os.path.join(os.path.dirname(__file__),
```

Comment:
Why include a binary file for testing? Can't this be done in the same manner as the other tests where data and metadata defined here is saved to a temp file? We then can't track any changes between binary file versions in git and it makes it clear to the next person reading this test file what data/metadata exactly is being tested.

Reply:
Because if it's written using the old writer, we can't write it each time with the new one.

Reply:
No we can't. Because we would be saving the file with the NEW metadata format, so we won't be able to test if we can read an old file with OLD metadata format, so any changes to the code would potentially break backwards compatibility and pass unnoticed.